public/js/src/module/records-queue.js

```diff
@@ -249,7 +296,7 @@ function uploadQueue() {
 }
 
 /**
- * Creates a zip file of all locally-saved records.
+ * Creates a zip file of all locally-saved records, except last-saved.
```

Comment:
and auto-saved?

Reply:
The original behavior did not filter out auto-saved. Should it?

Reply:
Oh! Interesting. How about being really explicit and saying "all draft records, an auto-saved record if there is one, and NOT the last-saved record (since it's redundant)."

There's some behavior here that would be good to have user-facing documentation around. As far as I know there is no Enketo user-facing documentation.

From what we've gleaned from the code, I believe that:
- auto save is on a per-form basis: there's one autosave record per form id (seems expected, same as Collect)
- auto save only happens when creating a new record (unexpected to me. Collect always autosaves even on record re-entry. Seems just as likely you'd want it then.)
- when there's an auto saved record for a form id and the user visits the form, the user is asked whether to load or discard the auto save (seems like a good interaction)
- there's no way to apply auto-saved values to a previously-saved-as-draft record (consistent with the second point above, as it needs to be)
- the auto save record is kept around until it's loaded and then saved or explicitly discarded (feels a little weird to me. I'd expect that if I load an auto saved record and then navigate to a previously-saved draft I'd be asked whether I want to save my record or proceed.)

Given all that, the auto-saved record being part of the export feels unexpected but not bad. It would feel more confusing/unexpected to me if there could be auto saves from opening existing draft records.

Reply:
How about this? "Creates a zip file of all locally-saved records, including drafts and auto-saved records, excluding last-saved."

Reply:
👍